### PR TITLE
Inserted CLI into toolset

### PR DESCRIPTION
**Pull request by dsplaisted**
This should include https://github.com/dotnet/cli/pull/9800, which should fix `dotnet --version`.

### CODE REVIEW

File: src/redist/redist.csproj [hunk 1]
```diff
@@ -164,18 +162,31 @@
...
   </Target>
+
+  <Target Name="CopyBundledVersions"
+          AfterTargets="Publish">
+
+    <Copy SourceFiles="$(PreviousStageBundledVersions)"
+          DestinationFolder="$(PublishDir)"/>
+    
+  </Target>
+
```
Comment (nguerrera) [inline src/redist/redist.csproj]: nit: extra newlines

File: build/RestoreDependency.proj [hunk 2]
```diff
@@ -8,13 +8,22 @@
...
+ -->
+
   <Target Name="CopySdkToOutput"
           DependsOnTargets="PrepareBundledDependencyProps;
                             EnsureDependencyRestored;
                             GetSdkItemsToCopy"
           Condition="'$(SdkLayoutDirectory)' != ''"
-          Inputs="@(SdkContent)"
-          Outputs="@(SdkContent->'$(SdkLayoutDirectory)/%(RecursiveDir)%(FileName)%(Extension)')">
+>
```
Comment (nguerrera) [inline build/RestoreDependency.proj]: Should we be using the arg to preserve newest here given that we can't use inputs/outputs.

Comment (dsplaisted) [reply build/RestoreDependency.proj]: The issue is we don't actually want to preserve newest here.  There may be newer versions of the files from a different source that we want to overlay with the right ones.

There's probably a better way to do this (perhaps exclude the SDK files from the initial files that are copied to the layout), but I didn't get into that in this PR.

**Comment by dsplaisted** [comment]
@dotnet/dnceng This leg failed because the machine ran out of disk space: https://ci.dot.net/job/dotnet_toolset/job/master/job/release_ubuntu_x64_prtest/35/

**Comment by dsplaisted** [comment]
@dotnet-bot test Ubuntu x64 Release Build

**Comment by MattGal** [comment]
@dsplaisted  ubuntu1404-20180321-7268e1 (the machine in question) has already been deleted, thanks for letting us know.

**Comment by dsplaisted** [comment]
@dotnet-bot test Ubuntu x64 Release Build

**Comment by dsplaisted** [comment]
@dotnet-bot test Ubuntu x64 Release Build

**Pull request merged**